CuratedContent/EconomyOfOpenSourceSW.md

```diff
@@ -0,0 +1,28 @@
+# A look at the economic forces in Open Source Software
+
+The Slate Article *The Internet Was Built on the Free Labor of Open Source Developers. Is That Sustainable?* looks at the economics driving the Open Source movement and how that is putting those utilizing that software without investing in it at risk.
```

Comment:
@curfman and @bernhold ... I recall @maherou talking about this at last ECP meeting but do we have a policty regarding line-breaks and/or approx. line length. Reason I ask, its difficult using GitHub's interface to highlight a particular concern in a line if that line spans many logical lines of the viewed article.

Reply:
The intro says its a *Slate* article but when I following the link, I wind up on a *Vice* site with a *Motherboard* tagline. I know nothing about any of these hosts/brands but was confused since they don't seem to line up. Is the reference to *Slate* even needed?

Can we make the article title a click-able URL ref?

Reply:
The reference to Slate came from some deeper reading and I should not have it here at all.  I can make the link active for the title.